content/plugins/gcp.md

```diff
@@ -278,7 +304,7 @@ mx:
 
 {{< /gsHighlight >}}
 
-The DNSRecord type can be connected to an instance or directly to an IP. In this case the (associated) public IP will be added to the list of resources.
+The DNSRecord type can be connected to an instance, or directly to an IP address. In the case of an IP address, the (associated) public IP is added to the list of resources.
```

Comment:
This is less correct than before. In *either* the `IPAddress` case *or* the `Instance` case the IP address (of the IPAddress or Instance) will be added to the list of resources (I agree the previous text was only *technically* more correct, while being more confusing :).

Perhaps... "The DNSRecord The DNSRecord type can be connected to a `cloudify.gcp.nodes.Instance`, or to a `cloudify.gcp.nodes.IPAddress`. Each instance or IP address node that is connected will have its public IP address added to the list of resources for this record." (feel free to revise further for clarity :)